codalab/apps/web/static/js/worksheet/table_item_interface.jsx

```diff
@@ -89,6 +89,7 @@ var TableItem = React.createClass({
                         <thead>
                             <tr>
                                 {header_html}
+                                <th className='table-column-_dropdown'></th>
```

Comment:
The rest of the tables entries are created dynamically with class names like 'table-column-uuid'.

If there are any columns with name 'dropdown' then we will have two columns with class 'table-column-dropdown' that is no good. So, we need to make a private css class name for cells with button.

There was a class 'table-column-_7C' in the less files before I had started to work on the project. I think it is for the same purpose. So, I decided to follow this convention and made class 'table-column-_dropdown' for the new column.